consensus/core/src/synchronizer.rs

```diff
@@ -597,7 +875,101 @@ mod tests {
                 sleep(latency).await;
             }
 
-            Ok(serialised)
+            Ok((serialised, vec![]))
+        }
+    }
+
```

Comment:
I intend to add more tests/refactor existing ones, but I'll do as follow up once get some comments